src/components/Dashboard/Dashboard.js

```diff
@@ -49,7 +49,7 @@ export class Dashboard extends Component {
 
             this.setHeaderbarVisibility()
```

Comment:
What does this do?  Shouldn't need to do anything with the headerbar in app platform...

Reply:
The print view doesn't show the headerbar:

![image](https://user-images.githubusercontent.com/6113918/92459963-a0654a00-f1c7-11ea-8c21-b6a29628104f.png)